tests/testthat/test_manifold.R

```diff
@@ -1 +1,26 @@
 # Unit tests for Manifold.
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+  rand <<- ceiling(10*runif(1))
+  manifold <<- Manifold$new(dimension = rand)
+
+
+
+test_that("testing dimension",{
+  TestDimension <- function(){
```

Comment:
The syntax of this test can be simplified by getting rid of the TestDimension function. More like what you did in 
 `expect_equal(euclidean.space$dimension,rand)`